clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -52,6 +52,10 @@
             {
                 "command": "broadcom-cobol-lsp.cpy-manager.edit-dataset-paths",
                 "title": "Edit dataset paths"
+            },
+            {
+                "command": "broadcom-cobol-lsp.cpy-manager.goto-settings",
+                "title": "Open Copybook location settings"
```

Comment:
Command name and title don't match.
The command named with `cpy-manager` part, but in `OpenSettingsCommand` `workbench.action.openSettings` executed with only `broadcom-cobol-lsp` as a parameter.
I see two options:
- change the title to "Open plugin settings" and command to "broadcom-cobol-lsp.goto-settings"
- change filtering string in `OpenSettingsCommand` to "broadcom-cobol-lsp.cpy-manager".